core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -67,7 +67,7 @@ public class Sep24Service {
   final Sep24TransactionStore txnStore;
   final EventService.Session eventSession;
   final InteractiveUrlConstructor interactiveUrlConstructor;
-  final MoreInfoUrlConstructor moreInfoUrlConstructor;
+  final MoreInfoUrlConstructor sep24moreInfoUrlConstructor;
```

Comment:
Just curious, why did we rename this field?

Reply:
It doesn't have to... I might have changed it because I'm splitting the 2 url constructor.